src/lightning_lite/plugins/collectives/collective.py

```diff
@@ -30,6 +30,10 @@ def rank(self) -> int:
     def world_size(self) -> int:
         ...
 
+    @property
+    def is_member(self) -> bool:
+        return self.rank >= 0
```

Comment:
IMO we shouldn't add this, this logic should be in whoever manages the collectives